nanoFramework.SignalR.Client/HubConnection.cs

```diff
@@ -312,7 +319,14 @@ private void InternalStart(bool reconnecting = false)
             else
             {
                 State = HubConnectionState.Disconnected;
-                throw new Exception("unable to connect to SignalR server");
+                if (string.IsNullOrEmpty(websocketException))
+                {
+                    throw new Exception("unable to connect to SignalR server");
+                }
+                else
+                {
+                    throw new Exception($"unable to connect to SignalR server, websocket connection failed with message: {websocketException}");
```

Comment:
We're getting a _bit_ too verbose here...
Flash space is at premium in MCUs, and with these messages we're blowing a lot of that.
Please rework this to make it as short as possible. Maybe something like "ERROR123" that allow to trace this to the exception in the code. 
Or if you think it's necessary, create a new exception type and use it here.

Reply:
Made the message shorter, tried to keep it meaningful. not being able to connect is most frustrating and common issue. This helps with support questions.